syntax = "proto2";

import "api/player_id.proto";
import "api/stats/team_stats.proto";

package api;

message Team {
    repeated api.PlayerId player_ids = 1;
    optional int32 score = 2;
    optional bool is_orange = 3;
    optional string name = 4;
    optional api.stats.TeamStats stats = 5;
}
